app/views/hosts/_form.html.erb

```diff
@@ -131,6 +136,9 @@
         <%= textarea_f f, :comment, :help_block => _("Additional information about this host"), :size => "col-md-8", :rows => "3", :class => "no-stretch" %>
       </div>
 
+      <% @tabs.values.each do |tab| %>
+        <%= tab %>
```

Comment:
Basically yes, the actual HTML. You can see the example of this usage here: https://github.com/theforeman/foreman/pull/2665/files#diff-b7708a9e000a6f55ba5cfb8553392dddR502
There is a difference between the construction of additional tabs (that do not require passing of `f` parameter, and aspect tabs. I am not sure if there could be other initialization process involved, so I left it open. I think your suggestion is a good idea, once we can agree on a common interface for all the tabs.